classes/Rabbit.py

```diff
@@ -0,0 +1,6 @@
+from classes.Animal import Animal
+
+
+class Rabbit(Animal):
+    def __init__(self, name, speed, step_interval):
+        super().__init__(name, speed, step_interval)
```

Comment:
this method could've been replaced with `pass`. 

this might also be a clue you're doing something wrong